packages/lab/src/calendar/useCalendar.ts

```diff
@@ -143,7 +143,7 @@ export function useCalendar(props: useCalendarProps) {
   const getInitialFocusedDate = (): DateValue => {
     const selectedDate = selectionManager.state.selectedDate;
     // Case range or offset
-    if (isRangeOrOffsetSelectionValue(selectedDate)) {
+    if (isRangeOrOffsetSelectionWithValues(selectedDate)) {
```

Comment:
Do we need to check if the endDate is present for this logic? What happens if you start a range selection then press tab?

Reply:
💥